server/webhook.go

```diff
@@ -18,31 +17,55 @@ type Payload struct {
 	Text     string `json:"text"`
 }
 
-func (s *Server) sendToWebhook(payload *Payload) error {
+func (s *Server) sendToWebhook(webhookURL string, payload *Payload) (*http.Response, error) {
+	err := validateSendToWebhookRequest(webhookURL, payload)
+	if err != nil {
+		badRequestR := &http.Response{
+			StatusCode: http.StatusBadRequest,
+			Body:       ioutil.NopCloser(bytes.NewBufferString(err.Error())),
+		}
```

Comment:
I am wondering the reasoning behind this? The body only contains the error string, and we are already returning the error. What extra are we including by injecting an artificially constructed response?

Same for others below.

Reply:
This does not appear to be resolved?